2996_reflection/reflection.md

```diff
@@ -7689,7 +7681,7 @@ template <class T>
 
 [#]{.pnum} *Constant When*:
 
-  - [#.#]{.pnum} `U` is a pointer type, `T` and `U` are similar types ([conv.qual]), and `is_convertible_v<U, T>` is `true`,
+  - [#.#]{.pnum} `U` is a pointer type, `T` and `U` are either similar or both function types, and `is_convertible_v<U, T>` is `true`,
```

Comment:
pointer to function types, presumably?

Reply:
@timsong-cpp I'll change to "both function pointer types" (since we use that phrase elsewhere too)